app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_post_likes_count
+
+  def update_post_likes_count
+    post.increment!(:likes_count)
+  end
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you made the `update_post_likes_count` a private method.